static/absolute/js/searchPage.js

```diff
@@ -68,7 +67,7 @@ window.onload = function () {
       alert("error posting to api");
       return;
     }
-
+    initMap();
```

Comment:
initMap() is being called twice in essentially the same place here.

The server emits a 'refresh' message using Socket.io in the post method whenever a user's information is added to Datastore. Below, that refresh message is received and searchRestaurants is called (and initMap() by extension). This allows the map to update automatically in real-time.

So searchRestaurants (and by extension, initMap) is being called in 2 places already: 
- When the search page is first loaded
- When new user information is submitted to Datastore

Reply:
Oh I see, i just pulled your changes. I was wondering if it would be better to separate init map from search restaurants?

Reply:
This is a good discussion you guys are having, as it contains some interesting architectural choices. I will let you guys discuss some more before weighing in.